components/fxa-client/src/oauth.rs

```diff
@@ -93,10 +93,21 @@ impl FirefoxAccount {
     /// * `scopes` - Space-separated list of requested scopes.
     /// * `wants_keys` - Retrieve scoped keys associated with scopes supporting it.
     pub fn begin_oauth_flow(&mut self, scopes: &[&str], wants_keys: bool) -> Result<String> {
-        let mut url = self.state.config.authorization_endpoint()?;
+        let mut url = if self.state.last_seen_profile.is_some() {
+            self.state.config.content_url_path("/oauth/force_auth")?
```

Comment:
:sadface: that we have to construct our own URL here, I wonder if it's worth moving this into the `Config` object in order to keep all URL-building in a single place.

Reply:
Yeah, a config for force_auth url might be useful

Reply:
Let's do this in a follow-up as we have constructed all our URLs in the code in this way.

Reply:
good point!

Reply:
https://github.com/mozilla/application-services/issues/1419